test/controller/pokemon.js

```diff
@@ -1240,8 +1243,8 @@ describe('PokemonController', () => {
       pkmnList = [];
       for (let i = 0; i < maxMultiUploadSize * 2; i++) {
         const res = await agent.post('/api/v1/pokemon')
-          // (this just creates a good mix of visibilities and boxes)
-          .field('box', i % 2 ? generalPurposeBox : unlistedBox)
+          .field('box', generalPurposeBox)
+          // (this just creates a good mix of visibilities)
```

Comment:
I decided to keep this clone list check to just private stuff, and do the unlisted stuff later.